[sdl2-mixer] CMake find_package is incomplete

**Describe the bug**
`find_package(SDL2_mixer REQUIRED)` fails, but sdl2-mixer is installed. The project builds fine with mingw (also version 2.6.1). 
Maybe related to #25284.

**Environment**
- Github runner: windows-2022
- Compiler: MSVC 19.32.31332.0

**Expected behavior**
`SDL2_MIXER_LIBRARIES` should be set.

**Failure logs**
After installation of sdl2-mixer:
```
 sdl2-mixer provides CMake targets:
    # this is heuristically generated, and may not be correct
    find_package(SDL2_mixer CONFIG REQUIRED)
    # note: 6 additional targets are not displayed.
    target_link_libraries(main PRIVATE FLAC::FLAC MPG123::mpg123 libxmp::libxmp tremor::tremor)
```
During cmake configuration to find package:
```
-- Found SDL2: C:/vcpkg/installed/x86-windows-static/lib/SDL2.lib (found version "2.0.22") 
-- Found SDL2main: C:/vcpkg/installed/x86-windows-static/lib/manual-link/SDL2main.lib (found version "2.0.22") 
-- Found SDL2_image: C:/vcpkg/installed/x86-windows-static/debug/lib/SDL2_image.lib (found version "2.0.5") 
CMake Error at C:/Program Files/CMake/share/cmake-3.23/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
  Could NOT find SDL2_mixer (missing: SDL2_MIXER_LIBRARIES) (found version
  "2.6.1")
Call Stack (most recent call first):
-- Configuring incomplete, errors occurred!
  C:/Program Files/CMake/share/cmake-3.23/Modules/FindPackageHandleStandardArgs.cmake:594 (_FPHSA_FAILURE_MESSAGE)
  cmake/Modules/FindSDL2_mixer.cmake:199 (FIND_PACKAGE_HANDLE_STANDARD_ARGS)
  C:/vcpkg/scripts/buildsystems/vcpkg.cmake:824 (_find_package)
  CMakeLists.txt:159 (find_package)
```

--- a/ports/sdl2-mixer/portfile.cmake
+++ b/ports/sdl2-mixer/portfile.cmake
@@ -36,4 +36,5 @@ vcpkg_fixup_pkgconfig()
 file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include")
 file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/share")
 
+file(INSTALL "${CMAKE_CURRENT_LIST_DIR}/usage" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}")
 file(INSTALL "${SOURCE_PATH}/LICENSE.txt" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}" RENAME copyright)

--- a/ports/sdl2-mixer/usage
+++ b/ports/sdl2-mixer/usage
@@ -1,3 +1,4 @@
 sdl2-mixer provides CMake targets:
+
     find_package(SDL2_mixer CONFIG REQUIRED)
     target_link_libraries($<IF:$<TARGET_EXISTS:SDL2_mixer::SDL2_mixer>,SDL2_mixer::SDL2_mixer,SDL2_mixer::SDL2_mixer-static>)

--- a/ports/sdl2-mixer/usage
+++ b/ports/sdl2-mixer/usage
@@ -1,0 +1,3 @@
+sdl2-mixer provides CMake targets:
+    find_package(SDL2_mixer CONFIG REQUIRED)
+    target_link_libraries($<IF:$<TARGET_EXISTS:SDL2_mixer::SDL2_mixer>,SDL2_mixer::SDL2_mixer,SDL2_mixer::SDL2_mixer-static>)

--- a/ports/sdl2-mixer/usage
+++ b/ports/sdl2-mixer/usage
@@ -1,4 +1,4 @@
 sdl2-mixer provides CMake targets:
 
     find_package(SDL2_mixer CONFIG REQUIRED)
-    target_link_libraries($<IF:$<TARGET_EXISTS:SDL2_mixer::SDL2_mixer>,SDL2_mixer::SDL2_mixer,SDL2_mixer::SDL2_mixer-static>)
+    target_link_libraries(main PRIVATE $<IF:$<TARGET_EXISTS:SDL2_mixer::SDL2_mixer>,SDL2_mixer::SDL2_mixer,SDL2_mixer::SDL2_mixer-static>)

--- a/ports/sdl2-mixer/vcpkg.json
+++ b/ports/sdl2-mixer/vcpkg.json
@@ -1,6 +1,7 @@
 {
   "name": "sdl2-mixer",
   "version": "2.6.1",
+  "port-version": 1,
   "description": "Multi-channel audio mixer library for SDL.",
   "homepage": "https://github.com/libsdl-org/SDL_mixer",
   "license": "Zlib",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -6498,7 +6498,7 @@
     },
     "sdl2-mixer": {
       "baseline": "2.6.1",
-      "port-version": 0
+      "port-version": 1
     },
     "sdl2-net": {
       "baseline": "2.0.1",

--- a/versions/s-/sdl2-mixer.json
+++ b/versions/s-/sdl2-mixer.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "6a3e42db7971d9ebe31a66c29968062d4a1db315",
+      "git-tree": "73638ee24d1e08340fcd3bd503f6e18a0a862cf9",
       "version": "2.6.1",
       "port-version": 1
     },

--- a/versions/s-/sdl2-mixer.json
+++ b/versions/s-/sdl2-mixer.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "73638ee24d1e08340fcd3bd503f6e18a0a862cf9",
+      "git-tree": "c06711b92dfd4c048f6a56c0236697012094a6e5",
       "version": "2.6.1",
       "port-version": 1
     },

--- a/versions/s-/sdl2-mixer.json
+++ b/versions/s-/sdl2-mixer.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "6a3e42db7971d9ebe31a66c29968062d4a1db315",
+      "version": "2.6.1",
+      "port-version": 1
+    },
+    {
       "git-tree": "7e802cb751276e60dc8046e2b6219dd9dae4c13a",
       "version": "2.6.1",
       "port-version": 0

